test/passing/module_item_spacing.ml

```diff
@@ -0,0 +1,71 @@
+[@@@ocamlformat "module-item-spacing=compact"]
+
+let z = this one is pretty looooooooooooooooooooooooooooooooooong
+let z = so is this oooooooooooooooooooooooooooooooooooooooooooone
+let f x = x + 1
+let z = this one is pretty looooooooooooooooooooooooooooooooooong
+let z = so is this oooooooooooooooooooooooooooooooooooooooooooone
+let g = ()
+
+let f = function
+  | `a | `b | `c -> foo
+  | `xxxxxxxxxxxxxxxxxx ->
+      yyyyyyyyyyyyyyyyyyyyyyyy
+        zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
+        zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
+        zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
+
+let x = 1
+let y = 2
+let z = this one is pretty looooooooooooooooooooooooooooooooooong
+let z = so is this oooooooooooooooooooooooooooooooooooooooooooone
+
+module A = AA
+module B = BB
+open AA
+module C = CC
+
+module M =
+  X
+    (Y)
+    (struct
+      let x = k
+    end)
+
+let x = 1
+let y = 2
+let x = 1
+let y = 2
+let c = {a : int; b : toto; c : char * char * char; d : [`Foo | `Bar]}
+let z = this one is pretty looooooooooooooooooooooooooooooooooong
+let z = so is this oooooooooooooooooooooooooooooooooooooooooooone
+
+type k = A | B | K of int * char * string | E
+
+let x = 1
+let z =
+  this
+    one
+    (is short)
+let y = 2
+let w = this one is toooooooooooooooooooooooooo (looooooooooooooooooooooooog but is (originally a one-liner))
+let k = z
+
+
```

Comment:
Do you expect this double empty line ?

Reply:
Could be due to `(fits_breaks ~force_fit_if:last_item "" "\n"))` line 3777 or so